### PR TITLE
add support for file I/O

### PR DESCRIPTION
**Pull request by rajarsheem**
sometimes we need to process multiple prompts from a file and generate multiple sequences. also, writing results to a file would be less verbose and fast.

### CODE REVIEW

**Comment by LysandreJik** [comment]
Hi @rajarsheem ! Thank you for your PR. With the example scripts, we are really reaching for **simple scripts that showcase how the library works** and how it interacts with different elements of the Pytorch codebase (ex: distributed learning, gradient clipping, ...).

Using a text file as input may be useful in some cases, however, I don't feel like it really gives a deeper understanding of the library, as it is just a different way to obtain a context string. I don't think it would be particularly worth it in terms of the added complexity/deeper understanding of the lib ratio.

Please don't let that discourage you from opening other PRs.
